apps/api/v2/src/ee/bookings/controllers/bookings.controller.ts

```diff
@@ -0,0 +1,112 @@
+import { CreateBookingInput } from "@/ee/bookings/inputs/create-booking.input";
+import { CreateReccuringBookingInput } from "@/ee/bookings/inputs/create-reccuring-booking.input";
+import { GetUser } from "@/modules/auth/decorators/get-user/get-user.decorator";
+import { AccessTokenGuard } from "@/modules/auth/guards/access-token/access-token.guard";
+import {
+  Controller,
+  Post,
+  Logger,
+  UseGuards,
+  Req,
+  InternalServerErrorException,
+  Body,
+  HttpException,
+} from "@nestjs/common";
+import { User } from "@prisma/client";
+import { Request } from "express";
+import { NextApiRequest } from "next/types";
+
+import { SUCCESS_STATUS } from "@calcom/platform-constants";
+import {
+  handleNewBooking,
+  BookingResponse,
+  HttpError,
+  handleNewRecurringBooking,
+  handleInstantMeeting,
+} from "@calcom/platform-libraries";
+import { ApiResponse } from "@calcom/platform-types";
+
+const handleBookingErrors = (err: Error | HttpError | unknown, type?: "recurring" | `instant`): void => {
```

Comment:
NIT: Can we define this as a function and move it under the controller so its at the bottom of file and controller at top for readability